node/src/neighborhood/overall_connection_status.rs

```diff
@@ -467,6 +507,21 @@ mod tests {
         );
     }
 
+    #[test]
+    fn can_receive_current_peer_addrs() {
+        let peer_1 = make_ip(1);
+        let peer_2 = make_ip(2);
+        let peer_3 = make_ip(3);
+
+        let subject = OverallConnectionStatus::new(vec![
+            make_node_descriptor(peer_1),
+            make_node_descriptor(peer_2),
+            make_node_descriptor(peer_3),
+        ]);
+
+        assert_eq!(subject.get_peer_addrs(), vec![peer_1, peer_2, peer_3]);
```

Comment:
`subject.get_peer_addrs()` is the Act part of this test. It should appear on a line by itself.